test/list_differ_test.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:observable/observable.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('$ListDiffer', () {
+    final diff = const ListDiffer<String>().diff;
```

Comment:
What about `const diff = ListDiffer<String>().diff`?

Reply:
Sadly this doesn't work because the ListDiffer object is const but its `diff` property is not.